utilities/patchOnUsBundles/script.js

```diff
@@ -0,0 +1,48 @@
+import { CosmosClient } from "@azure/cosmos";
+
+const key = "<cosmos-key>";
+const endpoint = "<cosmos-endpoint>";
```

Comment:
LGTM, could you only set the input for these two from env variable? To avoid sensible data in commits in the future